admin/sqlpatch.php

```diff
@@ -282,7 +283,6 @@ function executeSql($lines, $database, $table_prefix = '') {
           default:
             break;
           } //end switch
-//        } // endif $table_prefix

```

Comment:
It's commented out, so it's dead code.  It screws up bracket matching to have things like that.